src/style-spec/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mapbox/mapbox-gl-style-spec",
   "description": "a specification for mapbox gl styles",
-  "version": "13.26.0-dev",
```

Comment:
A note - we actually missed updating the previous dev version. 13.26.0 final is already released.